src/TabbedWebView.m

```diff
@@ -288,7 +288,7 @@ -(void)webView:(WebView *)sender decidePolicyForNavigationAction:(NSDictionary *
 			}
 		}
 	}
-	if (scheme == nil || [scheme isEqualToString:@""] || [scheme isEqualToString:@"http"] || [scheme isEqualToString:@"https"] || [scheme isEqualToString:@"feed"] || [scheme isEqualToString:@"file"] || [scheme isEqualToString:@"applewebdata"] || [scheme isEqualToString:@"about"])
+	if (scheme == nil || [scheme isEqualToString:@""] || [scheme isEqualToString:@"http"] || [scheme isEqualToString:@"https"] || [scheme isEqualToString:@"feed"] || [scheme isEqualToString:@"file"] || [scheme isEqualToString:@"applewebdata"] || [scheme isEqualToString:@"about"] || [scheme isEqualToString:@"data"])
```

Comment:
I know, try this then:
```Objective-C
NSArray *acceptedSchemes = @[@"", @"http", @"https"]; // ... feed, etc
if ([acceptedSchemes containsObject:scheme] || scheme == nil) {
    [listener use];
}
```
You can use the Xcode page guide at around 120 chars to help find code smells. If a line wraps over 120 chars then it is a good candidate for refactoring.